_data/projects/money-fox.yml

```diff
@@ -0,0 +1,13 @@
+name: Money Manager
+desc: "A way to make budgeting easy. A basic idea for the community, made even better by the community."
+site: https://github.com/Apply-Solutions/MoneyManager
+tags:
+- ".net"
+- Xamarin
+- C#
+- mono
+- Android
+- UWP
+upforgrabs:
+  name: easy
```

Comment:
Change this name  to `up-for-grabs` so that it matches up with the tag you use on the project...